### ISSUE TITLE
Minor Request

### ISSUE THREAD

**Issue by rvlewerissa** [pangloss/vim-javascript#1010]
Is it possible to enhance the syntax highlighting from:
<img width="466" alt="screen shot 2017-07-31 at 2 22 06 am" src="https://user-images.githubusercontent.com/15160023/28756377-32b6f530-7597-11e7-8ed1-34e396246e8c.png">

to:
<img width="451" alt="screen shot 2017-07-31 at 2 22 10 am" src="https://user-images.githubusercontent.com/15160023/28756378-32fda98a-7597-11e7-9b45-bb6eeac9e2b0.png">

Basically syntax for:
- property key for in object literal
- function key for in object literal
- function call

Thanks guys! :)

**Comment by amadeus** [comment]
We have a lot of that already, your colorscheme is just not attaching colors to it. You'll need to search around the web for a lil function to get the current highlight group under the cursor, then just set the colors you want

**Comment by juanibiapina** [comment]
Is there a repo somewhere with sane configs? Most colorschemes do not provide specifics for each language.
